src-vue/src/components/ThunderstoreModCard.vue

```diff
@@ -35,11 +35,30 @@
                 >
                     {{ modButtonText }}
                 </el-button>
-                <el-button link type="info" class="infoBtn" @click="openURL(mod.package_url)">
+
+                <!-- Information dropdown menu -->
+                 <el-button v-if="!modIsRemovable"
+                            link type="info" class="infoBtn" @click="openURL(mod.package_url)">
                     <el-icon>
                         <InfoFilled />
                     </el-icon>
                 </el-button>
+
+                <el-dropdown v-else>
+                    <el-icon class="infoBtn moreBtn">
+                        <MoreFilled />
+                    </el-icon>
+                    <template #dropdown>
+                        <el-dropdown-menu>
+                            <el-dropdown-item @click="openURL(mod.package_url)">
+                                More info
+                            </el-dropdown-item>
+                            <el-dropdown-item  @click="deleteMod(mod)">
```

Comment:
Don't you think an approval prompt is required here, just like local mods?

Reply:
Didn't you write that frontend code? :P

Anyway, it tried doing like 
```html
<el-dropdown-item>
    <el-popconfirm title="Are you sure to delete this mod?" @confirm="deleteMod(mod)">
        <template #reference>
            Remove mod  
        </template>
    </el-popconfirm>
</el-dropdown-item>
```

but that cause the dropdown menu to vanish the moment I hover the pop-up menu.

Demo:

https://user-images.githubusercontent.com/40122905/211149596-ee047a56-9d02-4c95-91c1-3ad490ef767f.mp4

Any ideas on how to resolve? ^^

Reply:
> Didn't you write that frontend code? :P

Hey, I can't do all your work 😄 

If popconfirm is not compatible with dropdown item, you can use a confirm message box: 
https://element-plus.org/en-US/component/message-box.html#confirm

Reply:
Okay, that worked ^^
Implemented in 16dd15ff7f9cebc82b9800c653c0d7b59a9ef2e9

Demo:
https://user-images.githubusercontent.com/40122905/211193481-79f84f51-6fae-4c52-9f5b-a45afd570db8.mp4

Reply:
Looks dope!
Maybe we should use this and [Message API](https://element-plus.org/en-US/component/message.html) more often in FlightCore :)

Reply:
Definitely ^^